src/prog_models/utils/calc_error.py

```diff
@@ -125,18 +127,19 @@ def RMSE(m, times, inputs, outputs, **kwargs):
     return np.sqrt(MSE(m, times, inputs, outputs, **kwargs))
 
 
-def MSE(self, times, inputs, outputs, **kwargs) -> float:
+def MSE(m, times, inputs, outputs, **kwargs) -> float:
```

Comment:
What was the reason to make this change?

Reply:
Other calc_error methods have defined the model to be 'm', so for the sake of consistency we changed 'self' to 'm'.

For a similar reason, I might have to include the '-> float' to all the calc_methods or maybe remove the '-> float'.

Reply:
ok, was just trying to understand if there was a reason for it. sounds fine. 

"For a similar reason, I might have to include the '-> float' to all the calc_methods or maybe remove the '-> float'." keep those for clarification. One of my requests for housekeeping is type hints. It's something that's inconsistent in ProgPy and will help future developers when contributing.